### PR TITLE
net-p2p/syncthing: bump to 0.14.10

### PR DESCRIPTION
**Pull request by sbraz**
Quite trivial: https://github.com/syncthing/syncthing/releases/tag/v0.14.10

### CODE REVIEW

**Comment by gktrk** [comment]
@gentoo/proxy-maint @djc

File: net-p2p/syncthing/syncthing-0.14.10.ebuild [hunk 1]
```diff
@@ -68,7 +68,7 @@ src_install() {
 		dobin bin/syncthing
 		exeinto /usr/libexec/syncthing
 		for exe in bin/* ; do
-			[ "${exe}" = "bin/syncthing" ] || doexe "${exe}"
+			[[ "${exe}" = "bin/syncthing" ]] || doexe "${exe}"
```
Comment (SoapGentoo) [inline net-p2p/syncthing/syncthing-0.14.10.ebuild]: `==` ?

File: net-p2p/syncthing/syncthing-0.14.10.ebuild [hunk 2]
```diff
@@ -68,7 +68,7 @@ src_install() {
 		dobin bin/syncthing
 		exeinto /usr/libexec/syncthing
 		for exe in bin/* ; do
```
Comment (SoapGentoo) [inline net-p2p/syncthing/syncthing-0.14.10.ebuild]: make `exe` local

**Comment by gktrk** [comment]
Merged in https://github.com/gentoo/gentoo/commit/63353de9cfe986a38d012943e4480e649f8fd06b. Thanks!

**Pull request closed without merging**